gpytorch/variational/tril_natural_variational_distribution.py

```diff
@@ -0,0 +1,137 @@
+import torch
+
+from ..distributions import MultivariateNormal
+from ..lazy import CholLazyTensor
+from .natural_variational_distribution import (
+    AbstractNaturalVariationalDistribution,
+    _NaturalToMuVarSqrt,
+    _phi_for_cholesky_,
+    _triangular_inverse,
+)
+
+
+class TrilNaturalVariationalDistribution(AbstractNaturalVariationalDistribution):
+    r"""A multivariate normal :obj:`~gpytorch.variational._VariationalDistribution`,
+    parameterized by the natural vector, and a triangular decomposition of the
+    natural matrix (which is not the Cholesky).
+
+    If the variational distribution is defined by :math:`\mathcal{N}(\mathbf m, \mathbf S)`, then
+    a :obj:`~gpytorch.variational.TrilNaturalVariationalDistribution` uses the parameterization:
+
+    .. math::
+
+        \begin{align*}
+            \boldsymbol \theta_\text{vec} &= \mathbf S^{-1} \mathbf m
+            \\
+            \boldsymbol \Theta_\text{tril\_mat} &= \text{cholesky}(\mathbf S)^{-1}
+        \end{align*}
+
+    The parameter :math:`\boldsymbol \Theta_\text{tril\_mat}`
+    (`natural_tril_mat`) is a lower triangular matrix.
+
+    The gradients with respect to the variational parameters calculated by this
+    class are instead the **natural** gradients. Thus, optimising its
+    parameters using gradient descent (:obj:`~torch.optim.SGDOptimizer`)
+    becomes natural gradient descent (see e.g. `Salimbeni et al., 2018`_).
+
+    .. seealso::
+        The `natural gradient descent tutorial
+        <examples/04_Variational_and_Approximate_GPs/Natural_Gradient_Descent.ipynb>`_
+        for use instructions.
+
+        The :obj:`~gpytorch.variational.NaturalVariationalDistribution`, which
+        needs less iterations to make variational regression converge, at the
+        cost of introducing numerical instability.
+
+    .. note::
+        The relationship of the parameter :math:`\boldsymbol \Theta_\text{tril\_mat}`
+        to the natural parameter :math:`\boldsymbol \Theta_\text{mat}` from
+        :obj:`~gpytorch.variational.NaturalVariationalDistribution` is
+        :math:`\boldsymbol \Theta_\text{mat} = -1/2 {\boldsymbol \Theta_\text{tril\_mat}}^T
+               {\boldsymbol \Theta_\text{tril\_mat}}`.
```

Comment:
The formatting of lines 50-51 in the readthedocs is all wrong. But `black` splits the line in two so I don't know what to do. Can I nest `.. note` and `.. math`? We'll see.

Reply:
You could also try replacing `boldsymbol` with `mathbf`? Might save a few characters?